Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -42,10 +42,19 @@ protected class MemoryCacheItem
             public BitmapImage Image { get; set; }
         }
 
+        private class ConcurrentTask
+        {
+            public Task<BitmapImage> Task { get; set; }
+
+            public bool IsAssureOnly { get; set; }
```

Comment:
I think instead of Assure we should be using Ensure as that would convey the right meaning